src/ui/writer.ts

```diff
@@ -72,7 +72,11 @@ export default class Writer {
     }
 
     public showError(message: string): void {
-        console.log(chalk.red(`❌ ${message}`))
+        if (this.headless) {
+            console.log('Error: ' + JSON.stringify(message))
```

Comment:
when you stringify it, the output has quotes around it so it's parsable by `JSON.parse`. Before it would output with no quotes so `JSON.parse` would throw. I could also make it an object if you think that makes more sense i.e `{ error: "some error" }`

Reply:
I think this is a good change to remove the emoji from the headless output. We could return the error as either an object or a string (although changing to an object could cause issues for any apps currently expecting a string). But stringifying the message that it already a string seems unnecessary, it would probably make more sense to update the extension logic to not attempt to parse the error as json

Reply:
ahh right. Sorry I misunderstood.